deepspeed/module_inject/replace_module.py

```diff
@@ -468,23 +429,24 @@ def replace_with_policy(child,
                     rotary_dim=rotary_dim,
                     mlp_after_attn=(rotary_dim is None or rotary_dim < 0),
                     mlp_act_func_type=policy.mlp_act_func_type,
-                    training_mp_size=training_mp_size,
+                    training_mp_size=config.training_mp_size,
                     bigscience_bloom=bigscience_bloom,
-                    max_out_tokens=max_out_tokens,
+                    max_out_tokens=config.max_out_tokens,
                     scale_attn_by_inverse_layer_idx=scale_attn_by_inverse_layer_idx)
 
-            if quantize and quantize_settings is not None:
+            if quantize and config.quantize_settings is not None:
                 (quantization_scales,
                  merge_count,
                  mlp_extra_grouping,
-                 quantize_groups) = quantize_settings
+                 quantize_groups) = config.quantize_settings
```

Comment:
As Mike mentioned above, I think this should refer to `config.quant` (or do we want to add an alias to `quantize_settings`?)